### PR TITLE
front:ensure right section is not there if stdcm fail

### LINKED ISSUE #5160: Modify error message string when STDCM times out
# Description: 

When a simulation is identified as not feasible by STDCM, present a message with the reason for rejection to the Railway Operator.

# Goal

To explain to Railway Operators the constraints on the network that lead to the simulation being knocked out. OSRD must provide at least the same information as the current tools. 

## AC

- If the algorithm does not find a solution, a message must be sent to users giving the reason for refusal: `Incompatibilité avec d'autres sillons`.

Fixed by #3972

### PR DESCRIPTION
**Pull request by alexandredamiron**
Just warn the user if stdcm request fail and keep the map

Capture d’écran 2023-04-20 à 17 52 47

### CODE REVIEW

File: front/public/locales/fr/stdcm.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "searchFailed": "Echec de la recherche de sillon"
+}
```
Comment (clarani) [inline front/public/locales/fr/stdcm.json]:
```suggestion
}

```
Missing new line

File: front/src/styles/scss/applications/_stdcm.scss [hunk 2]
```diff
@@ -6,3 +6,7 @@
     height: 40vh;
   }
 }
+
+.stdcm-error {
+  color: var(--danger);
+}
```
Comment (clarani) [inline front/src/styles/scss/applications/_stdcm.scss:22]:
```suggestion
}

```
Missing new line

**Review by clarani — APPROVED** [review]

**Review by Yohh — APPROVED** [review]
LGTM

**Review by clarani — CHANGES_REQUESTED** [review]
You need to squash your commits before merging

**Pull request closed without merging**